UnitTests/Sources/AttributedStringBuilderTests.swift

```diff
@@ -413,6 +413,14 @@ class AttributedStringBuilderTests: XCTestCase {
         XCTAssertEqual(numberOfBlockquotes, 3, "Couldn't find all the blockquotes")
     }
     
+    func testUserMentionAtachment() {
+        let string = "https://matrix.to/#/@test:matrix.org"
+        let attributedStringFromHTML = attributedStringBuilder.fromHTML(string)
+        XCTAssert(attributedStringFromHTML?.attachment.isNil == false)
```

Comment:
Should probably use the right assertion on these for better failure messages.
```suggestion
        XCTAssertNotNil(attributedStringFromHTML?.attachment)
```